combat-trainer.lic

```diff
@@ -3660,15 +3666,35 @@ class GameState
   include DRC
 
   $martial_skills = ['Brawling']
-  $melee_skills = ['Small Edged', 'Large Edged', 'Twohanded Edged', 'Small Blunt', 'Large Blunt', 'Twohanded Blunt', 'Staves', 'Polearms']
+  $edged_skills = ['Small Edged', 'Large Edged', 'Twohanded Edged']
+  $blunt_skills = ['Small Blunt', 'Large Blunt', 'Twohanded Blunt']
+  $staff_skills = ['Staves']
+  $polearm_skills = ['Polearms']
+  $melee_skills = $edged_skills + $blunt_skills + $staff_skills + $polearm_skills
   $thrown_skills = ['Heavy Thrown', 'Light Thrown']
   $twohanded_skills = ['Twohanded Edged', 'Twohanded Blunt']
   $aim_skills = ['Bow', 'Slings', 'Crossbow']
   $ranged_skills = $thrown_skills + $aim_skills
   $non_dance_skills = $ranged_skills + ['Brawling', 'Offhand Weapon']
-  $tactics_actions = %w[bob weave circle]
-
-  attr_accessor :mob_died, :last_weapon_skill, :danger, :parrying, :casting, :need_bundle, :cooldown_timers, :no_stab_current_mob, :loaded, :cast_timer, :casting_moonblade, :casting_weapon_buff, :use_charged_maneuvers, :casting_consume, :prepare_consume, :casting_cfb, :prepare_cfb, :wounds, :blessed_room, :currently_whirlwinding, :whirlwind_trainables, :reset_stance, :charges_total, :casting_sorcery, :hide_on_cast, :regalia_cancel, :last_regalia_type, :swap_regalia_type, :casting_regalia, :starlight_values, :casting_cyclic
+  $tactics_actions = ['bob', 'weave', 'circle']
+
+  attr_accessor :action_count, :aim_queue, :analyze_combo_array, :blessed_room, :cast_timer,
+                :casting, :casting_cfb, :casting_consume, :casting_cyclic, :casting_moonblade,
+                :casting_regalia, :casting_sorcery, :casting_weapon_buff, :charges, :charges_total,
+                :clean_up_step, :constructs, :cooldown_timers, :current_weapon_skill,
+                :currently_whirlwinding, :dance_queue, :dancing, :danger, :hide_on_cast,
+                :last_regalia_type, :last_weapon_skill, :loaded, :mob_died, :need_bundle,
+                :no_loot, :no_skins, :no_stab_current_mob, :no_stab_mobs, :parrying, :prepare_cfb,
+                :prepare_consume, :regalia_cancel, :reset_stance, :retreating, :starlight_values,
+                :swap_regalia_type, :target_weapon_skill, :use_charged_maneuvers,
+                :whirlwind_trainables, :wounds
+
+  attr_reader :aim_fillers, :aim_fillers_stealth, :ambush, :backstab, :balance_regen_threshold,
+              :cambrinth, :cambrinth_cap, :charged_maneuvers, :dance_actions, :dance_actions_stealth,
+              :dance_skill, :dance_threshold, :dedicated_camb_use, :dual_load, :fatigue_regen_threshold,
+              :ignored_npcs, :retreat_threshold, :stances, :stored_cambrinth, :summoned_weapons_element,
+              :summoned_weapons_ingot, :target_action_count, :target_increment, :use_analyze_combos,
+              :use_stealth_attacks, :use_weak_attacks, :weapons_to_train
```

Comment:
These were very long single-line declarations. I reformatted them and sorted them.

The lists were also spread out in multiple places in the GameState class (see old lines 4628-4629).

Now all in one spot.